imports/plugins/core/orders/server/no-meteor/resolvers/Refund/createdAt.js

```diff
@@ -0,0 +1,17 @@
+import moment from "moment";
+
+/**
+ * @name Refund/amount
+ * @method
+ * @memberof Order/GraphQL
+ * @summary converts the `createdAt` prop from a unix timestring to a format readable by `graphql-iso-data`
+ * @param {Object} refund - result of the parent resolver, which is a refund object in GraphQL schema format
+ * @return {String} A datetime string
+ */
+export default function createdAt(refund) {
+  // graphql-iso-data won't accept the Unix timestamp that is provided by Stripe
+  // we need to use moment to convert it to a readable format
+  const formattedDate = moment(refund.created).format();
+
+  return formattedDate;
+}
```

Comment:
Can we add a quick unit test for this? Almost trivial but easy to test. Any concerns about stripe specifics in a generic plugin?

Reply:
We could do it in the stripe plugin instead, however the `moment.format()` will accept other date formats as well and spit them out the same way, so although my comment and the reasoning behind this is because of stripe, this is really just making a consistent format for all dates that we get back.